### ISSUE TITLE
Variables ng_closed and wf_ng inside ndb.cutoff

### ISSUE THREAD

**Issue by sangallidavide** [yambo-code/yambo#75]
The I/O subroutine io_COL_CUT.F contains, in section 1, wf_ng and ng_closed.

This implies that the latter are changed when yambo finds the database, causing potential issues.

io_COL_CUT should contains a different variable ng_cutoff, eventually with checks about the size w.r.t wf_ng / ng_closed

**Comment by palful** [comment]
Can I just change

```
   call io_elemental(ID,I0=wf_ng,DB_I0=wf_ng_db,&
&       VAR=' RL components                   ',CHECK=.true.,OP=(/"<="/))
```

Into 

```
   call io_elemental(ID,I0=cutoff_ng,&
&       VAR=' RL components                   ',CHECK=.true.,OP=(/"<="/))
```
where `cutoff_ng` is declared in `mod_wave_func.F` and set by default to `wf_ng` during the setup (i.e. after the call to `io_db1`)?

**Comment by sangallidavide** [comment]
I think something like that would be ok. You would need to also define cutoff_ng_db

**Comment by palful** [comment]
Can you clarify why the argument `DB_I0=wf_ng_db` (or `cutoff_ng_db` with the new name) is needed for `io_elemental` in this case?

**Comment by sangallidavide** [comment]
In general for the cases in which `cutoff_ng_db > cutoff_ng`, it might happen that variables needs to be allocated with size `cutoff_ng_db` and later used with size `cutoff_ng`

See for example `src/io/io_GROT.F`

However here it is not used, so it is not strictly needed...

**Comment by sangallidavide** [comment]
This is now fixed (See merge #77 called `Issue #75`) I close it
